front-end/src/components/TranscriptUploadModal.jsx

```diff
@@ -40,7 +40,13 @@ class TranscriptUploadModal extends React.Component {
   }
 
   async handleUpload(file) {
-    TranscriptAPI.post(file);
+    const result = await TranscriptAPI.post(file);
+    if(result.status === 200){
+      this.props.toggleModal();
+    }
```

Comment:
We probably shouldn't just limit to 200, maybe test for 500, and then default to a close